portal/models/questionnaire_bank.py

```diff
@@ -322,6 +322,30 @@ def most_current_qb(user, as_of_date):
                     return last_found
         return last_found
 
+    @staticmethod
+    def indefinite_qb(user, as_of_date):
+        """Return namedtuple (QBD) for user representing their indefinite QB
+
+        The `indefinite` case is special.  Static method for this special case
+        as `most_current_qb()` handles all others.  Same return type, a QBD
+        named tuple.
+
+        :returns QBD: with values only if the user has an indefinite qb
+
+        """
+        indefinite_qb = QuestionnaireBank.qbs_for_user(user, 'indefinite')
+        if not indefinite_qb:
+            return QBD(None, None, None, None)
+
+        if len(indefinite_qb) > 1:
```

Comment:
Not a scenario we have to worry about right now (with only one indefinite QB in existence), but would it be better to just return the most recent (for example), and the log an error - similar to what we do for users belonging to more than one top-level org? Might help us avoid major crashes.

Reply:
err, most recent what?  :)

Reply:
Mm, maybe just `indefinite_qb[0]`? I'm just slightly worried about how much would break for this hypothetical future user if an error was thrown here. But maybe it's not something we need to worry about for now.